### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by loniceras**
Hi. Some new translations have been made in this PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,78 +52,78 @@
...
             {
                 "view": "bookmarksExplorer",
-                "contents": "In order to use Bookmarks, you have to open a folder or workspace first.\n[Open a Folder](command:_bookmarks.openFolderWelcome)\n[Open a Workspace](command:workbench.action.openWorkspace)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.1%",
                 "when": "workbenchState == empty"
             },
             {
                 "view": "bookmarksExplorer",
-                "contents": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.2%",
```
Comment (alefragnani) [inline package.json:68]: Avoid using `n` on ids. Use something meaningful, maybe based on the `when` clause, like `contents.whenNoFileOpen`

File: package.json [hunk 2]
```diff
@@ -52,78 +52,78 @@
...
                 "when": "workbenchState != empty && !editorIsOpen"
             },
             {
                 "view": "bookmarksExplorer",
-                "contents": "No bookmarks yet.\nIn order to use Bookmarks, place the cursor in any location in the file and run the command:\n[Bookmarks: Toggle](command:bookmarks.toggle)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.3%",
```
Comment (alefragnani) [inline package.json:73]: Avoid using `n` on ids. Use something meaningful, maybe based on the `when` clause, like `contents.whenHasFileOpen`

File: package.json [hunk 3]
```diff
@@ -52,78 +52,78 @@
...
                 "when": "workbenchState != empty && editorIsOpen"
             }
         ],
         "commands": [
             {
                 "command": "bookmarks.toggle",
                 "title": "%bookmarks.commands.toggle.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:81]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead.

File: package.json [hunk 4]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.jumpToNext",
                 "title": "%bookmarks.commands.jumpToNext.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:86]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 5]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.jumpToPrevious",
                 "title": "%bookmarks.commands.jumpToPrevious.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:91]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 6]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "_bookmarks.jumpTo",
                 "title": "%bookmarks.commands.jumpTo.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:96]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 7]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.selectLines",
                 "title": "%bookmarks.commands.selectLines.title%",
-                "category": "Bookmarks (Selection)"
+                "category": "%bookmarks.commands.category.2%"
```
Comment (alefragnani) [inline package.json:101]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks.selection%` instead.

File: package.json [hunk 8]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.expandSelectionToNext",
                 "title": "%bookmarks.commands.expandSelectionToNext.title%",
-                "category": "Bookmarks (Selection)"
+                "category": "%bookmarks.commands.category.2%"
```
Comment (alefragnani) [inline package.json:106]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks.selection%` instead.

File: package.json [hunk 9]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.expandSelectionToPrevious",
                 "title": "%bookmarks.commands.expandSelectionToPrevious.title%",
-                "category": "Bookmarks (Selection)"
+                "category": "%bookmarks.commands.category.2%"
```
Comment (alefragnani) [inline package.json:111]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks.selection%` instead.

File: package.json [hunk 10]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.shrinkSelection",
                 "title": "%bookmarks.commands.shrinkSelection.title%",
-                "category": "Bookmarks (Selection)"
+                "category": "%bookmarks.commands.category.2%"
```
Comment (alefragnani) [inline package.json:116]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks.selection%` instead.

File: package.json [hunk 11]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.list",
                 "title": "%bookmarks.commands.list.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:121]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 12]
```diff
@@ -52,78 +52,78 @@
...
             },
             {
                 "command": "bookmarks.toggleLabeled",
                 "title": "%bookmarks.commands.toggleLabeled.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:126]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 13]
```diff
@@ -137,7 +137,7 @@
             {
                 "command": "bookmarks.clear",
                 "title": "%bookmarks.commands.clear.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:140]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 14]
```diff
@@ -157,17 +157,17 @@
             {
                 "command": "bookmarks.listFromAllFiles",
                 "title": "%bookmarks.commands.listFromAllFiles.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:160]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 15]
```diff
@@ -157,17 +157,17 @@
...
             },
             {
                 "command": "bookmarks.clearFromAllFiles",
                 "title": "%bookmarks.commands.clearFromAllFiles.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:165]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 16]
```diff
@@ -157,17 +157,17 @@
...
             },
             {
                 "command": "bookmarks.whatsNew",
                 "title": "%bookmarks.commands.whatsNew.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:170]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 17]
```diff
@@ -180,7 +180,7 @@
             {
                 "command": "bookmarks.supportBookmarks",
                 "title": "%bookmarks.commands.supportBookmarks.title%",
-                "category": "Bookmarks"
+                "category": "%bookmarks.commands.category.1%"
```
Comment (alefragnani) [inline package.json:183]: Avoid using `n` on ids. Use `%bookmarks.commands.category.bookmarks%` instead

File: package.json [hunk 18]
```diff
@@ -381,8 +381,8 @@
                         "eachLineIndependently"
                     ],
                     "enumDescriptions": [
-                        "Creates bookmarks in all selected lines at once, if at least one of the lines don't have a bookmark",
-                        "Literally toggles a bookmark in each line, instead of making all lines equals"
+                        "%bookmarks.configuration.multicursor.toggleMode.enumDescriptions.1%",
+                        "%bookmarks.configuration.multicursor.toggleMode.enumDescriptions.2%"
```
Comment (alefragnani) [inline package.json:385]: Avoid using `n` on ids. Use something meaningful for each item.

File: package.json [hunk 19]
```diff
@@ -397,10 +397,10 @@
...
                     ],
                     "enumDescriptions": [
-                        "Don't use the selection (original behavior)",
-                        "Use the selected text (if available) directly, no confirmation required",
-                        "Suggests the selected text (if available). You still need to confirm.",
-                        "Suggests the selected text (if available) or the entire line (when has no selection). You still need to confirm."
+                        "%bookmarks.configuration.label.suggestion.enumDescriptions.1%",
+                        "%bookmarks.configuration.label.suggestion.enumDescriptions.2%",
+                        "%bookmarks.configuration.label.suggestion.enumDescriptions.3%",
+                        "%bookmarks.configuration.label.suggestion.enumDescriptions.4%"
```
Comment (alefragnani) [inline package.json:403]: Avoid using `n` on ids. Use something meaningful for each item.

File: package.nls.json [hunk 20]
```diff
@@ -1,6 +1,14 @@
...
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.2": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.3": "No bookmarks yet.\nIn order to use Bookmarks, place the cursor in any location in the file and run the command:\n[Bookmarks: Toggle](command:bookmarks.toggle)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.editor.context.label": "Bookmarks",
+    "bookmarks.editor.title.label": "Bookmarks",
+    "bookmarks.commands.category.1": "Bookmarks",
+    "bookmarks.commands.category.2": "Bookmarks (Selection)",
```
Comment (alefragnani) [inline package.nls.json:11]: Update the `n` ids to the new ids

File: package.nls.json [hunk 21]
```diff
@@ -34,7 +42,13 @@
     "bookmarks.configuration.showCommandsInContextMenu.description": "Specifies whether Bookmarks commands are displayed on the context menu",
     "bookmarks.configuration.sidebar.expanded.description": "Specifies wheher the Side Bar show be displayed expanded",
     "bookmarks.configuration.multicursor.toggleMode.description": "Specifies how multi cursor handles already bookmarked lines",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.1": "Creates bookmarks in all selected lines at once, if at least one of the lines don't have a bookmark",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.2": "Literally toggles a bookmark in each line, instead of making all lines equals",
```
Comment (alefragnani) [inline package.nls.json:46]: Update the `n` ids to the new ids

File: package.nls.json [hunk 22]
```diff
@@ -34,7 +42,13 @@
...
     "bookmarks.configuration.label.suggestion.description": "Specifies how labels are suggested when creating bookmarks",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.1": "Don't use the selection (original behavior)",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.2": "Use the selected text (if available) directly, no confirmation required",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.3": "Suggests the selected text (if available). You still need to confirm.",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.4": "Suggests the selected text (if available) or the entire line (when has no selection). You still need to confirm.",
```
Comment (alefragnani) [inline package.nls.json:51]: Update the `n` ids to the new ids

File: package.nls.pt-br.json [hunk 23]
```diff
@@ -1,6 +1,14 @@
...
     "bookmarks.views.Explorer.name": "Explorer",
+    "bookmarks.views.HelpAndFeedback.name": "Help and Feedback",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.1": "In order to use Bookmarks, you have to open a folder or workspace first.\n[Open a Folder](command:_bookmarks.openFolderWelcome)\n[Open a Workspace](command:workbench.action.openWorkspace)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.2": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.3": "No bookmarks yet.\nIn order to use Bookmarks, place the cursor in any location in the file and run the command:\n[Bookmarks: Toggle](command:bookmarks.toggle)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
```
Comment (alefragnani) [inline package.nls.pt-br.json:7]: Update the `n` ids to the new ids

File: package.nls.pt-br.json [hunk 24]
```diff
@@ -1,6 +1,14 @@
...
+    "bookmarks.editor.context.label": "Bookmarks",
+    "bookmarks.editor.title.label": "Bookmarks",
+    "bookmarks.commands.category.1": "Bookmarks",
+    "bookmarks.commands.category.2": "Bookmarks (Seleção)",
```
Comment (alefragnani) [inline package.nls.pt-br.json:11]: Update the `n` ids to the new ids

File: package.nls.pt-br.json [hunk 25]
```diff
@@ -34,7 +42,13 @@
     "bookmarks.configuration.showCommandsInContextMenu.description": "Define se os comandos Bookmarks são apresentados no menu de contexto",
     "bookmarks.configuration.sideBar.expanded.description": "Define se a Side Bar deve ser exibida expandida",
     "bookmarks.configuration.multicursor.toggleMode.description": "Define como funciona são tratadas as linhas que já possuem bookmark quando utiliza-se multicursor",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.1": "Creates bookmarks in all selected lines at once, if at least one of the lines don't have a bookmark",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.2": "Literally toggles a bookmark in each line, instead of making all lines equals",
```
Comment (alefragnani) [inline package.nls.pt-br.json:46]: Update the `n` ids to the new ids

File: package.nls.pt-br.json [hunk 26]
```diff
@@ -34,7 +42,13 @@
...
     "bookmarks.configuration.label.suggestion.description": "Define como rótulos são sugeridos quando se cria Bookmarks",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.1": "Don't use the selection (original behavior)",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.2": "Use the selected text (if available) directly, no confirmation required",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.3": "Suggests the selected text (if available). You still need to confirm.",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.4": "Suggests the selected text (if available) or the entire line (when has no selection). You still need to confirm.",
```
Comment (alefragnani) [inline package.nls.pt-br.json:51]: Update the `n` ids to the new ids

File: package.nls.pt-br.json [hunk 27]
```diff
@@ -1,6 +1,14 @@
 {
     "bookmarks.activitybar.title": "Bookmarks",
     "bookmarks.views.Explorer.name": "Explorer",
+    "bookmarks.views.HelpAndFeedback.name": "Help and Feedback",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.1": "In order to use Bookmarks, you have to open a folder or workspace first.\n[Open a Folder](command:_bookmarks.openFolderWelcome)\n[Open a Workspace](command:workbench.action.openWorkspace)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.2": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
```
Comment (alefragnani) [inline package.nls.pt-br.json:6]: Update the `n` ids to the new ids

File: package.nls.ru.json [hunk 28]
```diff
@@ -1,6 +1,14 @@
 {
     "bookmarks.activitybar.title": "Закладки",
     "bookmarks.views.Explorer.name": "Проводник",
+    "bookmarks.views.HelpAndFeedback.name": "Help and Feedback",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.1": "In order to use Bookmarks, you have to open a folder or workspace first.\n[Open a Folder](command:_bookmarks.openFolderWelcome)\n[Open a Workspace](command:workbench.action.openWorkspace)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.2": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
```
Comment (alefragnani) [inline package.nls.ru.json:6]: Update the `n` ids to the new ids

File: package.nls.ru.json [hunk 29]
```diff
@@ -1,6 +1,14 @@
...
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.3": "No bookmarks yet.\nIn order to use Bookmarks, place the cursor in any location in the file and run the command:\n[Bookmarks: Toggle](command:bookmarks.toggle)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
```
Comment (alefragnani) [inline package.nls.ru.json:7]: Update the `n` ids to the new ids

File: package.nls.ru.json [hunk 30]
```diff
@@ -1,6 +1,14 @@
...
+    "bookmarks.editor.context.label": "Закладки",
+    "bookmarks.editor.title.label": "Закладки",
+    "bookmarks.commands.category.1": "Закладки",
+    "bookmarks.commands.category.2": "Закладки (Выделение)",
```
Comment (alefragnani) [inline package.nls.ru.json:11]: Update the `n` ids to the new ids

File: package.nls.ru.json [hunk 31]
```diff
@@ -31,6 +39,16 @@
...
     "bookmarks.configuration.wrapNavigation.description": "Разрешить навигацию по первой и последней закладкам в области (текущий файл или все файлы)",
     "bookmarks.configuration.useWorkaroundForFormatters.description": "Использовать обходной путь для форматировщиков таких как Prettier, который не уведомляет об изменениях документа и путает поведение прикреплённой закладки",
+    "bookmarks.configuration.showCommandsInContextMenu.description": "Specifies whether Bookmarks commands are displayed on the context menu",
+    "bookmarks.configuration.sidebar.expanded.description": "Specifies wheher the Side Bar show be displayed expanded",
+    "bookmarks.configuration.multicursor.toggleMode.description": "Specifies how multi cursor handles already bookmarked lines",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.1": "Creates bookmarks in all selected lines at once, if at least one of the lines don't have a bookmark",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.2": "Literally toggles a bookmark in each line, instead of making all lines equals",
```
Comment (alefragnani) [inline package.nls.ru.json:46]: Update the `n` ids to the new ids

File: package.nls.ru.json [hunk 32]
```diff
@@ -31,6 +39,16 @@
...
+    "bookmarks.configuration.label.suggestion.description": "Specifies how labels are suggested when creating bookmarks",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.1": "Don't use the selection (original behavior)",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.2": "Use the selected text (if available) directly, no confirmation required",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.3": "Suggests the selected text (if available). You still need to confirm.",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.4": "Suggests the selected text (if available) or the entire line (when has no selection). You still need to confirm.",
```
Comment (alefragnani) [inline package.nls.ru.json:51]: Update the `n` ids to the new ids

File: package.nls.zh-cn.json [hunk 33]
```diff
@@ -1,6 +1,14 @@
 {
     "bookmarks.activitybar.title": "书签",
     "bookmarks.views.Explorer.name": "浏览器",
+    "bookmarks.views.HelpAndFeedback.name": "帮助和反馈",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.1": "为了使用书签，你需要打开一个文件夹或工作区。\n[打开文件夹](command:_bookmarks.openFolderWelcome)\n[打开工作区](command:workbench.action.openWorkspace)\n可以[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.2": "目前无书签。\n为了使用书签，你需要在编辑器中打开一个文件。\n[打开文件](command:workbench.action.quickOpen)\n可以[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.3": "目前无书签。\n为了使用书签，你需要将光标放在文件的任意位置，然后运行命令：\n[书签: 开关](command:bookmarks.toggle)\n可以[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
```
Comment (alefragnani) [inline package.nls.zh-cn.json:7]: Update the `n` ids to the new ids

File: package.nls.zh-cn.json [hunk 34]
```diff
@@ -1,6 +1,14 @@
...
+    "bookmarks.editor.context.label": "书签",
+    "bookmarks.editor.title.label": "书签",
+    "bookmarks.commands.category.1": "书签",
+    "bookmarks.commands.category.2": "书签（选择）",
```
Comment (alefragnani) [inline package.nls.zh-cn.json:11]: Update the `n` ids to the new ids

File: package.nls.zh-cn.json [hunk 35]
```diff
@@ -34,7 +42,13 @@
     "bookmarks.configuration.showCommandsInContextMenu.description": "允许在上下文菜单中显示书签命令（开关、跳至上一个、跳至下一个）。",
     "bookmarks.configuration.sidebar.expanded.description": "在书签侧边栏中默认显示展开的书签。",
     "bookmarks.configuration.multicursor.toggleMode.description": "指定多光标新建书签时对已有书签行的处理方式。",
-    "bookmarks.configuration.label.suggestion.description": "指定新建书签时建议标签的生成方式。",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.1": "如果至少有一行没有书签，则在所有选中的行创建书签。",
+    "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.2": "分别开关每一个选中的行的书签。",
```
Comment (alefragnani) [inline package.nls.zh-cn.json:46]: Update the `n` ids to the new ids

File: package.nls.zh-cn.json [hunk 36]
```diff
@@ -34,7 +42,13 @@
...
+    "bookmarks.configuration.label.suggestion.description": "指定新建含标签的开关时建议标签的生成方式。",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.1": "无建议的标签，需要手动输入标签。",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.2": "使用选择的文本作为标签，不需要额外的确认。",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.3": "使用选择的文本作为标签，需要额外的确认。",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.4": "使用选择的文本或（无选择的文本时）使用整个行作为标签，需要额外的确认。",
```
Comment (alefragnani) [inline package.nls.zh-cn.json:51]: Update the `n` ids to the new ids

**Review by alefragnani — CHANGES_REQUESTED** [review]
I've made some comments. Basically to replace the `n` named ids, to use something more meaningful.

**Comment by alefragnani** [comment]
Hi @loniceras ,

First of all, thank your for your PR 👍 

I've made some comments. Basically because of some of the ids are using a `n` based pattern. Please, update to something more meaningful, for easier understanding.

Thank you

**Comment by loniceras** [comment]
Hi! Thank you for your detailed comments. I'v made a revision based on these suggestions :)

**Comment by alefragnani** [comment]
Thank you for the updates. 

Everything appears to be ok. I’m planning to test it this weekend, and should be available in the next release.

Thank you

**Pull request merged**